app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::CommentsController < ApplicationController
+  class Api::CommentsController < ApplicationController
+    skip_before_action :verify_authenticity_token, only: [:create]
+    def index
+      @comments = Comment.where(post_id: params[:post_id])
+      render json: @comments, each_serializer: CommentSerializer
+    end
+
+    def create
+      @user = User.find(params[:user_id])
+      @post = @user.posts.find(params[:post_id])
+      @comment = @post.comments.build(comment_params)
+      @comment.author = @user
+      if @comment.save
+        render json: @comment, status: :created
+      else
+        render json: { errors: @comment.errors.full_messages }, status: :unprocessable_entity
+      end
+    end
```

Comment:
- [ ] Hi @JuanDBta, API end points could returns all posts for a user, all comments for a post as requested :+1: but I got an internal server error when I tried to create a comment for a post. I think you should look at it once more.